docs/references/javascript/organization/membership-request.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Organization Membership Request
+description: These are all methods on the Organization class that allow you to manage the membership requests for an organization.
+---
+
+# Organization Membership Request
+
+These are all methods on the [Organization](/docs/references/javascript/organization/organization) class that allow you to manage the membership requests for an organization.
+
+## `getMembershipRequests()`
+
+Retrieves a list of domains for the currently active organization.
+
+```typescript
+function getMembershipRequests(params?: GetMembershipRequestParams): Promise<PaginatedResources<OrganizationMembershipRequest>>;
+```
+
+### `GetMembershipRequestParams`
+
+| Name | Type | Description |
+| --- | --- | --- |
+| `initialPage?` | `number` | A number that can be used to skip the first n-1 pages. For example, if `initialPage` is set to 10, it is will skip the first 9 pages and will fetch the 10th page. |
+| `pageSize?` | `number` | A number that indicates the maximum number of results that should be returned for a specific page. |
+| `status?` | `'pending' \| 'accepted' \| 'revoked'` | The status an request can have. |
+
+### Returns
+
+| Type | Description |
+| --- | --- |
+| <code>Promise\<[PaginatedResponse][pag-res-ref]\<[OrganizationMembershipRequest][mem-request-ref]\>></code> | This method returns a `Promise` which resolves with a list of [`OrganizationMembershipRequest`][mem-request-ref] objects. |
```

Comment:
We are now deduping reference links ( see the [styleguide](https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md#13-de-dupe-reference-links) )

```suggestion
| Type | Description |
| --- | --- |
| <code>Promise\<[PaginatedResponse][pag-res-ref]\<[OrganizationMembershipRequest][mem-request-ref]\>></code> | This method returns a `Promise` which resolves with a list of `OrganizationMembershipRequest` objects. |
```